recipe/build.sh

```diff
@@ -1,6 +1,11 @@
 set -x
 set -o xtrace
 
+if [[ $target_platform  == osx-64 ]]; then
+  # https://github.com/TileDB-Inc/TileDB-R/issues/707
+  exit 0
+fi
+
```

Comment:
This code chunk is the reason the osx-64 build is failing. The package isn't being built, so then it fails during the test section. Please remove this entirely. From the build log, the osx-64 patch is being applied correctly, so this should fix the osx-64 build.